lms/djangoapps/courseware/courses.py

```diff
@@ -108,6 +108,18 @@ def get_course_by_id(course_key, depth=0):
         raise Http404(u"Course not found: {}.".format(six.text_type(course_key)))
 
 
+def check_if_course_org_in_site_org_filters(course_key):
+    """
+    Given a course key, checks if the course org is in site org.
+
+    If the course is not in site orgs, raises a 404.
+    """
+    current_site_orgs = configuration_helpers.get_current_site_orgs()
+    course = get_course_by_id(course_key)
+    if course.org not in current_site_orgs:
+        raise Http404(u"Course not found: {}.".format(six.text_type(course_key)))
```

Comment:
Can we move this code to edly app.

Reply:
As we discussed, I have changed it to use the `is_course_org_same_as_site_org` from edly app